ga4gh/datamodel/variants.py

```diff
@@ -415,6 +427,28 @@ def convertVariant(self, record, callSetIds):
             sampleIterator += 1  # REMOVAL
         return variant
 
+    def getVariant(self, compoundId):
+        if compoundId.chrom in self._chromFileMap:
+            varFileName = self._chromFileMap[compoundId.chrom]
+        else:
+            raise exceptions.ObjectNotFoundException(compoundId)
+        pos = int(compoundId.pos)
+        referenceName, startPosition, endPosition = \
+            self.sanitizeVariantFileFetch(
+                compoundId.chrom, pos, None)
+        # TODO vcf.fetch's startPosition arg seems to be exclusive,
```

Comment:
@jeromekelleher we're not taking this into account in our interval iteration algorithm, are we?